opendistro-for-elasticsearch/opendistro-build
# [HELM] Kibana pods annotations

*Description of changes:*
User can define pods annotations for Kibana

*Test Results:*

* Deploy using the following chart values :
```
kibana:
  podAnnotations:
    vault.security.banzaicloud.io/vault-addr: "https://vault:8200"
```
* Check the annotations on the Kibana pod


By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
